src/components/modes/teacher/Settings.js

```diff
@@ -84,7 +86,7 @@ const Settings = () => {
           onClose={handleModalClose}
           className={classes.modal}
         >
-          <div className={classes.modalContainer}>
+          <div id="settings" className={classes.modalContainer}>
```

Comment:
We usually use a `selectors.js` to define ids and classnames related to tests. That way, you can refer to the same strings instead of hardcoding them. https://github.com/graasp/graasp-app-sketchfab/blob/master/src/config/selectors.js